eng/imports/HostAgnostic.props

```diff
@@ -9,7 +9,6 @@
     <!-- Infrastructure -->
     <!-- Path Property: PkgCodecov -->
     <PackageReference Include="Codecov"                             ExcludeAssets="all" GeneratePathProperty="true" />
-    <PackageReference Include="MicroBuild.Core.Sentinel"                  PrivateAssets="all" />
```

Comment:
Why are we removing this reference? What does it do currently?

Reply:
I am not sure what it did but it is not required anymore. We have other MicroBuild packages which brings in whatever we need to build. My assumption at this point is this was needed for microbuild to function few years ago when our build infra was different. (That package was last modified in 2019)